ferveo-python/ferveo/__init__.pyi

```diff
@@ -190,3 +189,75 @@ def decrypt_with_shared_secret(
         dkg_params: DkgPublicParameters,
 ) -> bytes:
     ...
+
+
+class ThresholdEncryptionError(Exception):
+    pass
+
+
+class InvalidShareNumberParameter(Exception):
+    pass
+
+
+class InvalidDkgStateToDeal(Exception):
+    pass
+
+
+class InvalidDkgStateToAggregate(Exception):
+    pass
+
+
+class InvalidDkgStateToVerify(Exception):
+    pass
+
+
+class InvalidDkgStateToIngest(Exception):
+    pass
+
+
+class DealerNotInValidatorSet(Exception):
+    pass
+
+
+class UnknownDealer(Exception):
+    pass
+
+
+class DuplicateDealer(Exception):
+    pass
+
+
+class InvalidPvssTranscript(Exception):
+    pass
+
+
+class InsufficientTranscriptsForAggregate(Exception):
+    pass
+
+
+class InvalidFinalKey(Exception):
```

Comment:
Since `dkg.final_key` got changed to `dkg.public_key` should we still use "FinalKey" in the exception?